docker_cli/commands.py

```diff
@@ -0,0 +1,143 @@
+import os
+from shlex import split
+from subprocess import DEVNULL, PIPE, CalledProcessError, run
+from typing import Iterable, List, Union
+
+from ._docker_mamba import mamba_lockfile_command
+from ._exceptions import ImageNotFoundError
+from ._image import Image
+from ._utils import is_conda_pkg_name, universal_tag_prefix
+
+
+def dropin(tag: str) -> None:
+    """
+    Initiates a drop-in session on an image.
+
+    Parameters
+    ----------
+    tag : str
+        The tag or ID of the image.
+    """
+    try:
+        image: Image = Image(tag)
+    except ImageNotFoundError as err:
+        print(f"Error: {str(err)}")
+        exit(1)
+
+    image.drop_in()
+    print("Drop-in session ended.")
+    exit()
+
+
+def remove(
+    tags: Iterable[str],
+    force: bool = True,
+    quiet: bool = False,
+    ignore_prefix: bool = False,
+) -> None:
+    """
+    Remove all Docker images that fit a given tag or wildcard.
+
+    This tag or wildcard will take the form [UNIVERSAL PREFIX]-[tag or wildcard] if the
+    prefix does not already fit this.
+
+    Parameters
+    ----------
+    force : bool, optional
+        Whether or not to force the removal. Defaults to True.
+    tags : Iterable[str]
+        An iterable of tags or wildcards to be removed.
+    quiet : bool, optional
+        Whether or not to run quietly. Defaults to False.
+    ignore_prefix: bool, optional
+        Whether or not to ignore the universal prefix and only use the tag or wildcard.
+        Use with caution, as this will remove ALL images matching the wildcard.
+        e.g. remove(["*"], ignore_prefix = True) will remove all images.
```

Comment:
You can enclose code snippets in double backticks (``` `` ```) in your docstrings in order to use a monospaced font.

https://numpydoc.readthedocs.io/en/latest/format.html#common-rest-concepts

```suggestion
        e.g. ``remove(["*"], ignore_prefix = True)`` will remove all images.
```